### ISSUE TITLE
new feature request: Early stopping on the validation test

### ISSUE THREAD

**Issue by helenlly** [intel-analytics/analytics-zoo#3000]
refer to https://groups.google.com/g/bigdl-user-group/c/7CavjVEEekE

" Is there currently a way to define an early stopper trigger on the validation set with NNClassifier model ?
I mean trigger wich stops the training when the loss function goes below a certain value.
To my knowledge .setEndWhen() method works exclusively on the training set and not on the validation set.
Is there a plan to develop the same method on the validation set ? "
